code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/implant/carrion_spider/explosive
 	name = "explosive spider"
-	desc = "A spider that's about to blow the dust."
+	desc = "A spider that's bulgy, like it's about to burst. Better not put it in your pants."
```

Comment:
What the f

Reply:
oh wait i missed the perfect opportunity